Sources/Testing/Support/CustomIssueRepresentable.swift

```diff
@@ -12,7 +12,7 @@
 /// record themselves as test issues.
 ///
 /// When a type conforms to this protocol, values of that type can be passed to
-/// ``Issue/record(_:_:)``. The testing library then calls the
+/// ``Issue/record(_:_:_:)``. The testing library then calls the
```

Comment:
Was this reference incorrect even before the recent change? The last parameter should've been `sourceLocation:` so now it should be
```suggestion
/// ``Issue/record(_:severity:sourceLocation:)``. The testing library then calls the
```